diff --git a/node_modules/react-native-gifted-chat/lib/GiftedChat.js.flow b/node_modules/react-native-gifted-chat/lib/GiftedChat.js.flow
index 414eefc..7cf98e3 100644
--- a/node_modules/react-native-gifted-chat/lib/GiftedChat.js.flow
+++ b/node_modules/react-native-gifted-chat/lib/GiftedChat.js.flow
@@ -100,6 +100,7 @@ export type GiftedChatProps<TMessage: IMessage = IMessage> = $ReadOnly<{|
   renderDay?: (DayProps<TMessage>) => React.Node,
   renderTime?: (TimeProps<TMessage>) => React.Node,
   renderFooter?: () => React.Node,
+  renderChatEmpty?: () => React.Node,
   renderChatFooter?: () => React.Node,
   renderInputToolbar?: InputToolbarProps => React.Node,
   renderComposer?: ComposerProps => React.Node,
