diff --git a/node_modules/react-native-gifted-chat/lib/Day.js b/node_modules/react-native-gifted-chat/lib/Day.js
index 7829a72..afc54a0 100644
--- a/node_modules/react-native-gifted-chat/lib/Day.js
+++ b/node_modules/react-native-gifted-chat/lib/Day.js
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React, { PureComponent } from 'react';
 import { StyleSheet, Text, View, } from 'react-native';
-import dayjs from 'dayjs';
+import dayjs from '../../../lib/utils/day';
 import Color from './Color';
 import { StylePropType, isSameDay } from './utils';
 import { DATE_FORMAT } from './Constant';
@@ -26,9 +26,7 @@ export default class Day extends PureComponent {
             return (<View style={[styles.container, containerStyle]}>
           <View style={wrapperStyle}>
             <Text style={[styles.text, textStyle]}>
-              {dayjs(currentMessage.createdAt)
-                .locale(this.context.getLocale())
-                .format(dateFormat)}
+              {dayjs(currentMessage.createdAt).format(DATE_FORMAT)}
             </Text>
           </View>
         </View>);
diff --git a/node_modules/react-native-gifted-chat/lib/GiftedChat.js b/node_modules/react-native-gifted-chat/lib/GiftedChat.js
index 73ba4ee..e2c5947 100644
--- a/node_modules/react-native-gifted-chat/lib/GiftedChat.js
+++ b/node_modules/react-native-gifted-chat/lib/GiftedChat.js
@@ -4,8 +4,7 @@ import { Platform, StyleSheet, View, SafeAreaView, KeyboardAvoidingView, } from
 import { ActionSheetProvider, } from '@expo/react-native-action-sheet';
 import uuid from 'uuid';
 import { getBottomSpace } from 'react-native-iphone-x-helper';
-import dayjs from 'dayjs';
-import localizedFormat from 'dayjs/plugin/localizedFormat';
+import dayjs from '../../../lib/utils/day';
 import * as utils from './utils';
 import Actions from './Actions';
 import Avatar from './Avatar';
@@ -23,7 +22,6 @@ import Send from './Send';
 import Time from './Time';
 import GiftedAvatar from './GiftedAvatar';
 import { MIN_COMPOSER_HEIGHT, MAX_COMPOSER_HEIGHT, DEFAULT_PLACEHOLDER, TIME_FORMAT, DATE_FORMAT, } from './Constant';
-dayjs.extend(localizedFormat);
 class GiftedChat extends React.Component {
     constructor(props) {
         super(props);
diff --git a/node_modules/react-native-gifted-chat/lib/Time.js b/node_modules/react-native-gifted-chat/lib/Time.js
index 24b06cc..6943370 100644
--- a/node_modules/react-native-gifted-chat/lib/Time.js
+++ b/node_modules/react-native-gifted-chat/lib/Time.js
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
-import dayjs from 'dayjs';
+import dayjs from '../../../lib/utils/day';
 import Color from './Color';
 import { TIME_FORMAT } from './Constant';
 import { StylePropType } from './utils';
diff --git a/node_modules/react-native-gifted-chat/lib/utils.js b/node_modules/react-native-gifted-chat/lib/utils.js
index cf91b98..a6f0345 100644
--- a/node_modules/react-native-gifted-chat/lib/utils.js
+++ b/node_modules/react-native-gifted-chat/lib/utils.js
@@ -1,5 +1,5 @@
 import PropTypes from 'prop-types';
-import dayjs from 'dayjs';
+import dayjs from '../../../lib/utils/day';
 export const StylePropType = PropTypes.oneOfType([
     PropTypes.array,
     PropTypes.object,
