diff --git a/node_modules/react-native/.DS_Store b/node_modules/react-native/.DS_Store
new file mode 100644
index 0000000..44dca2d
Binary files /dev/null and b/node_modules/react-native/.DS_Store differ
diff --git a/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.h b/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.h
index 02de122..6c24aa0 100644
--- a/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.h
+++ b/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.h
@@ -1,34 +1,27 @@
-/*
- * Copyright (c) Facebook, Inc. and its affiliates.
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
 #import <UIKit/UIKit.h>
-#if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= 12000) /* __IPHONE_12_0 */
-#import <UIKit/UIUserActivity.h>
-#endif
+
 #import <React/RCTEventEmitter.h>
 
 @interface RCTLinkingManager : RCTEventEmitter
 
-+ (BOOL)application:(nonnull UIApplication *)app
-            openURL:(nonnull NSURL *)URL
-            options:(nonnull NSDictionary<UIApplicationOpenURLOptionsKey, id> *)options;
++ (BOOL)application:(nullable UIApplication *)app
+            openURL:(nullable NSURL *)URL
+            options:(nullable NSDictionary<UIApplicationOpenURLOptionsKey,id> *)options;
 
-+ (BOOL)application:(nonnull UIApplication *)application
-              openURL:(nonnull NSURL *)URL
-    sourceApplication:(nullable NSString *)sourceApplication
-           annotation:(nonnull id)annotation;
++ (BOOL)application:(nullable UIApplication *)application
+            openURL:(nullable NSURL *)URL
+  sourceApplication:(nullable NSString *)sourceApplication
+         annotation:(nullable id)annotation;
 
-+ (BOOL)application:(nonnull UIApplication *)application
-    continueUserActivity:(nonnull NSUserActivity *)userActivity
-      restorationHandler:
-        #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= 12000) /* __IPHONE_12_0 */
-            (nonnull void (^)(NSArray<id<UIUserActivityRestoring>> *_Nullable))restorationHandler;
-        #else
-            (nonnull void (^)(NSArray *_Nullable))restorationHandler;
-        #endif
++ (BOOL)application:(nullable UIApplication *)application
+continueUserActivity:(nullable NSUserActivity *)userActivity
+  restorationHandler:(nullable void (^)(NSArray * __nullable))restorationHandler;
 
 @end
diff --git a/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.mm b/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.mm
index 0f14b40..5a5657f 100644
--- a/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.mm
+++ b/node_modules/react-native/Libraries/LinkingIOS/RCTLinkingManager.mm
@@ -75,12 +75,7 @@ + (BOOL)application:(UIApplication *)application
 
 + (BOOL)application:(UIApplication *)application
 continueUserActivity:(NSUserActivity *)userActivity
-  restorationHandler:
-    #if defined(__IPHONE_OS_VERSION_MAX_ALLOWED) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= 12000) /* __IPHONE_12_0 */
-        (nonnull void (^)(NSArray<id<UIUserActivityRestoring>> *_Nullable))restorationHandler {
-    #else
-        (nonnull void (^)(NSArray *_Nullable))restorationHandler {
-    #endif
+  restorationHandler:(nullable void (^)(NSArray * __nullable))restorationHandler {
   if ([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
     NSDictionary *payload = @{@"url": userActivity.webpageURL.absoluteString};
     [[NSNotificationCenter defaultCenter] postNotificationName:kOpenURLNotification
diff --git a/node_modules/react-native/scripts/.packager.env b/node_modules/react-native/scripts/.packager.env
new file mode 100644
index 0000000..361f5fb
--- /dev/null
+++ b/node_modules/react-native/scripts/.packager.env
@@ -0,0 +1 @@
+export RCT_METRO_PORT=8081
